### PR TITLE
Update readme to reflect current state

### PR DESCRIPTION
**Pull request by Jkrzy**
For #182, adding additional content to the `README.md`. 

Highlights include details and limitations of local develop, guidance for collaborators, and referring feedback/commends to the existing google form in place for findtreatmentbeta.18f.gov.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,82 @@
...
+
+Run the following commands from wihtin the directory where this file, `README.md`, is located.
 
-## Setup local development
-Follow these steps to checkout `samhsa-prototype` in your local environment
 ```
-git clone git@github.com:18F/samhsa-prototype.git
-cd samhsa-prototype
 npm install
+npm start
```
Comment (hursey013) [inline README.md]: There is also `npm run dev` currently which combines the above two commands, but the above accomplishes the same result.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,82 @@
...
 ----
-Working with SAMHSA to prototyping a new user experience for finding treatment.
+[18F](https://18f.gsa.gov) is collaborating with [SAMHSA](https://www.samhsa.gov) to develop a new experience for finding treatment. This repository contains the development for the front-end application of that experience.
+
+## Instances
+`master` is our primary and stable development branch which is available at https://findtreatmentbeta.18f.gov
+
+## Feedback
+
+If you'd like to provide feedback or comments on the website please use [this google form](https://docs.google.com/forms/d/e/1FAIpQLSfJWR3ZmRXfVwlqn5Oq_TSn3jeav9RW6J3K0Dcyk_t_hp7ICw/viewform). This repository is currently only used for tracking internal development of the website.
```
Comment (hursey013) [inline README.md]: So if folks from the public find a technical issue with the site they should not use github issues?

Comment (Jkrzy) [reply README.md]: Good catch, let's differentiate.

**Comment by Jkrzy** [comment]
Made two tweaks stemming for your comments, thanks @hursey013!

**Pull request merged**